### PR TITLE
Result detail

### PR DESCRIPTION
**Pull request by jlhogan**

### CODE REVIEW

File: src/components/ResultDetail/BasicResultDetail.tsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    {result.website && (
+      <ResultDatum Icon={Website}>
+        <Heading className="usa-sr-only">Website</Heading>
+        <ExternalLink target="#" href={result.website}>{result.website}</ExternalLink>
+      </ResultDatum>
+    )}  
+    {result.address?.length && (
+      <ResultDatum Icon={Location}>
+        <Heading className="usa-sr-only">Address</Heading>
+        <div>{result.address.join('')}</div>
```
Comment (cathydeng) [inline src/components/ResultDetail/BasicResultDetail.tsx:41]: think we'll have to do `.join(", ")`

but also, if we aren't displaying addresses on multiple lines anywhere (not sure how the real data looks in a single line, like if theres weird wrapping and we'd prefer two lines) then we should update this in the data processing script

File: src/components/ResultDetail/Hours.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import { HoursOfOperation } from "../../types";
+
+type HoursProps = {
```
Comment (cathydeng) [inline src/components/ResultDetail/Hours.tsx:3]: nit: we can add this to types.ts as `WeeklyHoursOfOperation` and update the type for `CareProvider.hours` there, to minimize duplication w/ laying out days of the week

File: src/components/ResultDetail/Hours.tsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    thursday: HoursOfOperation;
+    friday: HoursOfOperation;
+    saturday: HoursOfOperation;
+  }
+}
+
+function Hours({ hours }: HoursProps) {
+  return (
+    <>
+      <div className="margin-bottom-1">Monday: {hours.monday ? `${hours.monday.start} - ${hours.monday.end}`: 'Closed'}</div>
```
Comment (cathydeng) [inline src/components/ResultDetail/Hours.tsx:18]: one open data Q: if hours of operation are missing, i'm not sure if it means "Closed" or hours are unknown

File: src/components/ResultDetail/ResultDatum.tsx [hunk 4]
```diff
@@ -0,0 +1,17 @@
+import { Grid } from "@trussworks/react-uswds";
+import React, { PropsWithChildren } from "react";
+
+type ResultDatumProps = {
+  Icon: React.FunctionComponent<React.SVGProps<SVGSVGElement>>
+}
+
+function ResultDatum( { Icon, children }: PropsWithChildren<ResultDatumProps>) {
+  return (
+    <Grid row className="margin-y-2">
```
Comment (cathydeng) [inline src/components/ResultDetail/ResultDatum.tsx:10]: oo neat didn't know about this `margin-y` pattern

**Review by cathydeng — APPROVED** [review]
just some lil things, LGTM!

File: src/components/ResultDetail/Hours.tsx (hunk 2 above)
Comment (jlhogan) [reply src/components/ResultDetail/Hours.tsx:3]: yes, will do in PR with remainder of this work

File: src/components/ResultDetail/BasicResultDetail.tsx (hunk 1 above)
Comment (jlhogan) [reply src/components/ResultDetail/BasicResultDetail.tsx:41]: I think we should make a note to revisit how we might want/need this info. for the google maps direction link, it's just URL encoded string. Not sure there's anywhere we need separate data, i could imagine a world where bucketing by zip would be desired but not at present

**Pull request merged**